src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,533 @@
+
+
+def setup_market(faucet_client: FaucetClient, viewer: EconiaViewer) -> int:
+    lot_size = 10 ** (18 - 3)  # tETH has 18 decimals, want 1/1000th granularity
+    tick_size = 10 ** (6 - 3)  # tUSDC has 6 decimals, want 1/1000th granularity
+    min_size = 6
+    market_id = get_market_id_base_coin(
+        viewer, COIN_TYPE_ETH, COIN_TYPE_USDC, lot_size, tick_size, min_size
+    )
+    if market_id == None:
+        account_XCH = Account.generate()
+        faucet_client.fund_account(account_XCH.address(), 1 * (10**8))
+        faucet_client.fund_account(account_XCH.address(), 1 * (10**8))
+        faucet_client.fund_account(account_XCH.address(), 1 * (10**8))
+        faucet_client.fund_account(account_XCH.address(), 1 * (10**8))
+        faucet_client.fund_account(account_XCH.address(), 1 * (10**8))
+        print("Market does not exist yet, creating one...")
+        calldata = register_market_base_coin_from_coinstore(
+            ECONIA_ADDR,
+            TypeTag(StructTag.from_str(COIN_TYPE_ETH)),
+            TypeTag(StructTag.from_str(COIN_TYPE_USDC)),
+            TypeTag(StructTag.from_str(COIN_TYPE_APT)),
+            lot_size,
+            tick_size,
+            min_size,
+        )
+        EconiaClient(NODE_URL, ECONIA_ADDR, account_XCH).submit_tx_wait(
+            calldata
+        )
+        market_id = get_market_id_base_coin(
+            viewer, COIN_TYPE_ETH, COIN_TYPE_USDC, lot_size, tick_size, min_size
+        )
+        events = get_market_registration_events(viewer)
+        report_market_creation_event(
+            list(filter(lambda e: e["data"]["market_id"] == market_id, events))[
+                0
+            ]
+        )
+    if market_id == None:
+        exit()
+    print(f"Market ID: {market_id}")
+    return market_id
+
+
+def get_best_prices(
+    viewer: EconiaViewer, market_id: int
+) -> Tuple[Optional[int], Optional[int]]:
+    price_levels = get_price_levels(viewer, market_id)
+
+    price_bid = None
+    if len(price_levels["bids"]) != 0:
+        price_bid = price_levels["bids"][0]["price"]
+
+    price_ask = None
+    if len(price_levels["asks"]) != 0:
+        price_ask = price_levels["asks"][0]["price"]
+
+    return price_bid, price_ask
+
+
+def report_best_price_levels(viewer: EconiaViewer, market_id: int):
+    price_levels = get_price_levels(viewer, market_id)
+    if len(price_levels["bids"]) == 0 and len(price_levels["asks"]) == 0:
+        print("There is no tETH being bought or sold right now!")
+        return
+
+    if len(price_levels["bids"]) != 0:
+        best_bid_level = price_levels["bids"][0]
+        best_bid_level_price = best_bid_level["price"]  # in ticks
+        best_bid_level_volume = best_bid_level["size"]  # in lots
+        print(
+            f"Market SELL: {best_bid_level_price} ticks per lot / {best_bid_level_volume} available lots"
+        )
+    else:
+        print("There are no lots of tETH being BOUGHT right now")
```

Comment:
Can you change this to say there are no open bids?

Reply:
will do